content/en/platform/corda/5.1/application-networks/creating/members/cpi.md

```diff
@@ -17,7 +17,8 @@ This section describes how to build a {{< tooltip >}}member{{< /tooltip >}} {{<
 2. [Generate the Group Policy File]({{< relref "#generate-the-group-policy-file" >}})
```

Comment:
we need to update 5.2 docs as well afaik

Reply:
updated 5.2